src/worker/generate-kmp.ts

```diff
@@ -24,9 +24,7 @@ export async function generateKmp(
 ): Promise<ArrayBuffer> {
   const kmpJsonFile = generateKmpJson({
     languages: [{ name: langName, id: bcp47Tag }],
-    // TODO: change generateKmpJson() to accept a modelID argument instead of splitting it here:
-    authorID: modelId.split(".")[0],
-    modelTag: modelId.split(".")[2],
+    modelId: modelId,
```

Comment:
Might want to make it ID to keep it consistent

Reply:
Have we decided to use which pattern? modelId or modelID? I found in different files we use different patterns 😂  @eddieantonio

Reply:
After the informal Slack poll, I decided we're sticking to `modelID`, even though we used `modelId` in the past. This is now part of [the new conventions document](https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#general-keep-capitals-intact-in-names-with-acronyms-and-initialisms-in-camelcase)! 🥳